.changeset/perfect-elephants-do.md

```diff
@@ -0,0 +1,21 @@
+---
+'@backstage/plugin-catalog-backend': patch
```

Comment:
Could also be `minor` since the processor is exported. But I don't think anyone uses that as is always brought by the catalog builder.